vassal-app/src/main/java/VASSAL/launch/PlayerWindow.java

```diff
@@ -166,14 +166,15 @@ public void actionPerformed(ActionEvent e) {
     toolsMenu.add(debugCheckbox);
     DebugControls.setCheckBox(debugCheckbox);
 
-    try {
-      final URL url = new File(Documentation.getDocumentationBaseDir(),
-                               "README.html").toURI().toURL();
-      mm.addAction("General.help", new ShowHelpAction(url, null));
-    }
-    catch (MalformedURLException e) {
-      ErrorDialog.bug(e);
-    }
+    // This file doesn't have particularly useful information, and the weird entry ("Help"???) was cluttering the Help menu.
+    //try {
+    //  final URL url = new File(Documentation.getDocumentationBaseDir(),
+    //                           "README.html").toURI().toURL();
+    //  mm.addAction("General.help", new ShowHelpAction(url, null));
+    //}
+    //catch (MalformedURLException e) {
+    //  ErrorDialog.bug(e);
+    //}
```

Comment:
Is there any reason to leave this code here?

Reply:
Nope. Just removed it.